pkg/util/bandwidth/linux_test.go

```diff
@@ -270,14 +270,14 @@ func TestFindCIDRClass(t *testing.T) {
 			}
 			if test.expectNotFound {
 				if found {
-					t.Errorf("unexpectedly found an interface: %s %s", class, handle)
+					t.Errorf("unexpectedly found an interface: %s", classAndHandle)
 				}
 			} else {
-				if class != test.expectedClass {
-					t.Errorf("expected: %s, found %s", test.expectedClass, class)
+				if classAndHandle[0][0] != test.expectedClass {
```

Comment:
Only `classAndHandle[0]` is needed here. Why should we return `[][]string{}` for function `findCIDRClass `? No need IMO.

Reply:
As mentioned above, there is a need for [][]